src/test/clojure/xtdb/test_json.clj

```diff
@@ -102,7 +104,10 @@
        (check-arrow-json-file expected actual)
 
        (.endsWith file-name ".transit.json")
-       (check-transit-json-file expected actual)))))
+       (check-transit-json-file expected actual)
+
+       (.endsWith file-name ".binpb")
+       (= (seq (Files/readAllBytes expected)) (seq (Files/readAllBytes actual)))))))
```

Comment:
These changes have nothing todo with `json`. We could add some utilities for `binpb` if we would like to look at the `binpb`. I think we would need to know what kind of protobuf we are talking about.

Reply:
binpbs aren't necessarily ordered, they're a map (same as Transit) so need to be parsed to check for equality. The objects you get back after parsing are comparable, so I think it'd be a case of passing the parsing fn through (which you can now do with Clojure 1.12)

reckon it's best we split this out into `check-pbuf`?